Makefile

```diff
@@ -42,12 +44,12 @@ test-reconciler:
 # Requires ko 0.2.0 or newer.
 test-images:
 	for img in $(TEST_IMAGES); do \
-		ko resolve --tags=latest -RBf $$img ; \
+		KO_DOCKER_REPO=$(DOCKER_REPO_OVERRIDE) ko resolve --tags=$(TEST_IMAGE_TAG) $(KO_FLAGS) -RBf $$img ; \
```

Comment:
why move this here?

Reply:
For consistency with the other components.

Reply:
@matzew @yselkowitz refers to e.g. https://github.com/openshift/knative-serving/blob/main/Makefile#L39 , it's not necessary but fine, IMO.